docker-compose.yml

```
@@ -0,0 +1,31 @@
+version: "3.9"
+
+services:
+
+  model-service:
+
+    build: ../model-service
+
+    container_name: model-service
+
+    ports:
+      - "8081:8081"
+
+    environment:
+      MODEL_SERVICE_PORT: 8081
+
```

At the end of each, I also suggest adding a [restarting policy](https://docs.docker.com/engine/containers/start-containers-automatically/), as mentioned in the grading rubric, something along these lines:

```suggestion
    restart: unless-stopped
```